apps/api_web/lib/api_web/controllers/route_controller.ex

```diff
@@ -13,7 +13,8 @@ defmodule ApiWeb.RouteController do
 
   @filters ~w(id stop type direction_id date)
   @pagination_opts [:offset, :limit, :order_by]
-  @includes ~w(stop)
+  @includes_show ~w(stop)
+  @includes_index ~w(line) ++ @includes_show
```

Comment:
This is the right idea, but backwards: `line` is the only include for `show`, and `stop` is also available for `index`.

Reply:
Yes, just fixed it